### DISCUSSION TITLE
KSADS PTSD Questions

### DISCUSSION THREAD

**Discussion by magcanady** [now-i-know-my-abcd/docs#67] [Where in the World is…? (Help)]
Hi all!

I have a question about the ways the parent-report KSADS PTSD questions are asked in the 2year and 4year follow-ups (ksads_ptsd_raw_754_p through ksads_ptsd_raw_770_p). Specifically, what is unit of measurements are the questions are capturing (e.g., since the last time you completed this survey, in the past 6 months, etc)?

Thank you so much!

**Comment by laikevin128** [comment]
Based on the questions, I'm guessing the unit of measurement is in a lifetime. But since these questions are proprietary, you can contact KSADS for more specifics https://www.ksadslogin.net/ksads-comp/ContactUs.aspx
